lib/insights/api/common/open_api/generator.rb

```diff
@@ -562,6 +564,16 @@ def handle_custom_route_action(_route_action, _verb, _primary_collection)
           def schema_overrides
             {}
           end
+
+          def unique_name(operation_id)
+            if @operation_ids.include?(operation_id)
+              # Method names would be aa, aa_1, aa_2 so we dont mangle the first aa
+              counter = @operation_ids.select { |name| name.start_with?(operation_id) }.count
```

Comment:
@mkanoor Does this run in console, like a rake task?  Does it make sense to report that we are handling this conflict resolution to stdout instead of silently handling it?

Reply:
@gmcculloug I have posted an example of the exception that is raised.